lib/Sender.js

```diff
@@ -78,7 +78,7 @@ class Sender {
    */
   doPing (data, options) {
     var mask = options && options.mask;
-    this.frameAndSend(0x9, data || '', true, mask);
+    this.frameAndSend(0x9, data ? Buffer.from(data.toString()) : null, true, mask);
```

Comment:
Is `toString()` to make sure that whatever is passed to `ping` is converted to a string?